.helm/assure-hmrc-data/values/uat.yaml

```diff
@@ -23,6 +23,7 @@ ingress:
   annotations:
     external-dns.alpha.kubernetes.io/set-identifier: "assure-hmrc-data-laa-assure-hmrc-data-uat-green"
     external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/whitelist-source-range: <%= ENV['ALLOW_LIST'] %>
```

Comment:
we may not want to whitelist UAT (so we don't have to be on the VPN for it)

Reply:
👍 had added uat for testing purposes but now uat should not have allow listing in place